pkg/sql/colexec/cast_test.go

```diff
@@ -184,7 +184,7 @@ func BenchmarkCastOp(b *testing.B) {
 						}
 						selectivity := selectivity
 						if !useSel {
-							selectivity = 1.0
+							selectivity = 0
```

Comment:
Doesn't this mean that no tuples will be selected?

Reply:
No, it's the opposite. The higher the selectivity, the fewer tuples are selected. The code also has a special case for 0 which is no set selection vector.

Reply:
True, the comment is wrong then.